src/components/common/TmSelectNetwork.vue

```diff
@@ -45,10 +45,8 @@ export default {
       }
     }
   },
-  watch: {
-    networks: networks => {
-      this.updateSelectedNetwork(networks)
-    }
+  mounted() {
+    this.updateSelectedNetwork(this.networks)
```

Comment:
This will work but the better fix and to keep this async is to use `immediate: true` in the watcher. Please search the repo for it

Reply:
oki

Reply:
It doesn't work 🤷‍♀